### DISCUSSION TITLE
Browser

### DISCUSSION THREAD

**Discussion by Ezra-Lawrence** [vhqtvn/VHSCode-Android#180] [Ideas]
Could you add some sort of browser inside the app that can be used  with live service and also has the inspector elements dev tools

**Comment by vhqtvn** [comment]
It may not be simple, i'll look into it later.

**Comment by sdproyi** [comment]
I fixed that issue with downloading a extention named live server after that I opened my chrome and added a js code into my html that helps me to see the console 
![Screenshot_20220921_191944_vn vhn vsc](https://user-images.githubusercontent.com/71813700/191595710-d052610a-9899-431e-92d7-9ca62d76c86d.jpg)
`html
<script src="//cdn.jsdelivr.net/npm/eruda" </script>
<script>eruda.init();</script>
`

**Reply by Ezra-Lawrence** [reply]
How did you open the browser on the side

**Reply by HyperKiko** [reply]
split screen
